.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java_version: [11,17,21]
+        java_version: [11,17,21,22-ea]
```

Comment:
There is the separate jdk-ea build so I don't know why we are adding 22-ea here? I don't think we want this here.

Reply:
the EA build is currently running with JDK 23-ea (hence why it is failing)